livekit-agents/livekit/agents/ipc/proc_pool.py

```diff
@@ -14,6 +14,8 @@
     "process_created", "process_started", "process_ready", "process_closed"
 ]
 
+MAX_CONCURRENT_INITIALIZATIONS = 3
```

Comment:
I'm not exposing this value for now (simplicity)

Reply:
It generally happens at startup. Otherwise it happens when you start to receive more jobs than the num_idle_processes